src/al.jl

```diff
@@ -41,11 +45,15 @@ function al(nlp :: AbstractNLPModel; max_iter :: Int = 1000, max_time :: Real =
   gp = zeros(T, nlp.meta.nvar)
   Jx = jac(nlp, x)
   fx, gx = objgrad(nlp, x)
+  lvar = eltype(nlp.meta.lvar) == T ? nlp.meta.lvar : T.(nlp.meta.lvar)
+  uvar = eltype(nlp.meta.uvar) == T ? nlp.meta.uvar : T.(nlp.meta.uvar)
 
   # penalty parameter
   μ = T.(10.0)
   # Lagrange multiplier
-  y = T.(cgls(Jx', gx)[1])
+  y = with_logger(subsolver_logger) do
```

Comment:
Why not use the µ stored in the model two lines above?

Reply:
This is the starting `mu`.

Reply:
Ah ok, the AL model hasn't been created yet. But shouldn't µ be a parameter?

Reply:
You're right, since we're using `al_nlp.x` and `al_nlp.cx`, it makes sense to use `al_nlp.mu`.